sdk/src/wallet/migration/migrate_3.rs

```diff
@@ -83,6 +73,20 @@ impl Migration<crate::client::stronghold::StrongholdAdapter> for Migrate {
     }
 }
 
+fn migrate_client_options(client_options: &mut serde_json::Value) -> Result<()> {
+    if let Some(params) = client_options
+        .get_mut("protocolParameters")
+        .map(|p| p.as_object_mut().unwrap())
```

Comment:
Does this have 0 chance to panic ever? Otherwise maybe use and_then instead of map